src/main/java/com/hms_networks/sc/cumulocity/CConnectorMain.java

```diff
@@ -81,6 +94,9 @@
   /** Connector control tag object for accessing its value. */
   private static TagControl connectorControlTag = null;
 
+  /** Connector measurement enable control tag object for accessing its value. */
+  private static TagControl connectorMeasurementEnableControlTag = null;
```

Comment:
TagControl needs a try/catch so it can't be in-lined

Reply:
Correct. This relies on logging which in turn relies on the log level from the configuration file being loaded. This can only occur during start up/initialization.